Added "aggressiv" category and extended others with some very important emoticons

rokk4 opened the pull request.

rokk4 committed:
Update weechataboo.scm

rokk4 committed:
Update weechataboo.scm

oxzi commented on weechataboo.scm:
```diff
@@ -75,25 +75,28 @@
          ("smug"  "(￣ω￣),(￣ー￣),（￣ー￣）,(^～^)")
-         ("wink"  "ヾ(＾∇＾),ヾ(☆▽☆),(。-ω-)ﾉ,( ･ω･)ﾉ")))
+         ("wink"  "ヾ(＾∇＾),ヾ(☆▽☆),(。-ω-)ﾉ,( ･ω･)ﾉ")
+         ("aggressiv" "o(-`д´- ｡),凸ಠ益ಠ)凸,' ̿'̵͇̿̿з=(◕_◕)=ε/̵͇̿̿/'̿'̿ ̿,O=('-'Q),｡゜(｀Д´)゜｡,┌∩┐(ಠ_ಠ)┌∩┐")
```
Please rename *aggresiv* to *aggressive*.

oxzi commented on weechataboo.scm:
```diff
@@ -75,25 +75,28 @@
   (let*
     ; Some defaults which may be useful‥
     ((emotions
-       '(("angry" "눈_눈,(¬_¬),(｀ε´),(¬▂¬),（▽д▽）")
```
Could you please indent all emotion-strings to the depth of the longest category name (*aggressive* in this case) and resort the list alphabetically based on the categories.

```scheme
; Like this
("aggressive" "…")
("angry"      "…")
("blush"      "…")
```

Just saw that I already had a wrong indenting with *excited*. This makes this PR even more important :^)

oxzi commented on weechataboo.scm:
```diff
@@ -75,25 +75,28 @@
+         ("wink"  "ヾ(＾∇＾),ヾ(☆▽☆),(。-ω-)ﾉ,( ･ω･)ﾉ")
+         ("aggressiv" "o(-`д´- ｡),凸ಠ益ಠ)凸,' ̿'̵͇̿̿з=(◕_◕)=ε/̵͇̿̿/'̿'̿ ̿,O=('-'Q),｡゜(｀Д´)゜｡,┌∩┐(ಠ_ಠ)┌∩┐")
+         ("cat" "≋≋≋≋≋̯̫⌧̯̫(ˆ•̮ ̮•ˆ)")
+         ("sword" "╰(◕ヮ◕)つ¤=[]———,╰(⇀︿⇀)つ-]═───,∩(˵☯‿☯˵)つ¤=[]:::::>)))
```
Please add closing quotes at the string's end.

geistesk commented:
Thanks for your pull request!
I'd be happy to merge this one after applying the requested changes. Could you also squash these commits into one and give it a more significant name.

rokk4 committed:
Applied requested changes for the pull request

Intended the category list to the longest item.
Category names are now in  alphabetic order.
Added missing closing quote to string.

rokk4 committed:
really did the right indentation now

linebreak from small sized window fucked it up the first time, lel

oxzi committed:
Ordered defaults alphabetically

geistesk commented:
Thanks for your work!
